### DISCUSSION TITLE
Editing Tree view components in itwin model

### DISCUSSION THREAD

**Discussion by ShivaGugulothu** [iTwin/community#26] [Q&A]
Can any one help regarding how to edit Tree view components in itwin  model

**Comment by saskliutas** [comment]
Hi, can you provide more details? What do you want to achieve? Are you trying to create your own Tree component or customize the one of the existing?

**Comment by ShivaGugulothu** [comment]
Creating own tree component or customizing the existing  both can solve my problem. How can i achieve it?

**Reply by grigasp** [reply]
have a look at this: https://itwinui.bentley.com/docs/tree
